modules/gcs/README.md

```diff
@@ -50,7 +50,7 @@ module "kms" {
     bucket_key = {
       iam_bindings = {
         bucket_key_iam = {
-          members = ["serviceAccount:${module.project.service_accounts.robots.storage}"]
+          members = [module.project.service_agents.storage.iam_email]
```

Comment:
I guess we need to add `services = ["storage.googleapis.com"]` above for this grant to work.

Otherwise tests complain:
```
Error: Unsupported attribute

  on main.tf line 18, in module "kms":
  18:           members = [module.project.service_agents.storage.iam_email]
    ├────────────────
    │ module.project.service_agents is object with 2 attributes

This object does not have an attribute named "storage".
```